.rubocop_todo.yml

```diff
@@ -1,16 +1,115 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2022-11-26 07:45:38 UTC using RuboCop version 1.22.3.
+# on 2025-03-31 17:33:57 UTC using RuboCop version 1.75.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# It may be possible for us to use safe_load, but we'd have to pass the
-# safelists, like `whitelist_classes` into our serializer, and the serializer
-# interface is a public API, so that would be a breaking change.
+# Offense count: 23
+# Configuration parameters: EnforcedStyle, AllowedGems, Include.
+# SupportedStyles: Gemfile, gems.rb, gemspec
+# Include: **/*.gemspec, **/Gemfile, **/gems.rb
+Gemspec/DevelopmentDependencies:
```

Comment:
I think worth fixing all the new offences in this file or disabling some cops if they aren't preferable.

Reply:
sure, any preference here? i was trying to keep this as safe as possible but i'm happy to fix what i can (or just disable the rules).

Reply:
i just re-ran the unsafe fixes on everything under `spec` since that should be relatively safe to do. will look at the non-auto-fixable items next

Reply:
ok - i've fixed or ignored everything _except_ the last two. both of these could have consequences beyond what our test suite covers so i'd suggest we defer those to separate prs?